### DISCUSSION TITLE
Procedure of learning

### DISCUSSION THREAD

**Discussion by Neha-maheen25** [github/india#871] [Students]
Hello ! I am new to this intership and I don't know how to begin this internship nd the time period in which we should complete the task ..i have doubt if ..we can listen to the lectures whenever we need and the time stops when we discontinue or does it keeps going on ..and i have my university exams going on from the August to the end of August ..does this intership contains assignments to be done every week ?

**Comment by vmskhan** [comment]
https://www.youtube.com/watch?v=RZlRsekXqI4
watch this video for answers to your questions @Neha-maheen25
